src/AutoRest.CSharp/Mgmt/Models/MgmtClientOperation.cs

```diff
@@ -164,7 +156,7 @@ private IReadOnlyList<Parameter> EnsureMethodParameters()
             if (_extensionParameter is not null)
                 parameters.Add(_extensionParameter);
             if (IsLongRunningOperation)
-                parameters.Add(WaitForCompletionParameter);
+                parameters.Add(KnownParameters.WaitForCompletion);
             var overrideParameters = OperationMappings.Values.First().OverrideParameters;
             if (overrideParameters.Length > 0)
             {
```

Comment:
We are also using a `CancellationTokenParameter`, could we also take this opportunity to switch to the one in `src/AutoRest.CSharp/Common/Output/Models/Shared/KnownParameters.cs` as well?